Repository: plone.app.testing


Branch: refs/heads/master
Date: 2019-02-07T18:13:09+01:00
Author: ale-rt (ale-rt) <alessandro.pisa@gmail.com>
Commit: https://github.com/plone/plone.app.testing/commit/560b69dbb8a2d10f585b2319e2221145835aa175

WIP Workaround for https://github.com/plone/Products.CMFPlone/issues/2734

Refs. https://github.com/plone/Products.CMFPlone/issues/2734

Files changed:
M src/plone/app/testing/layers.rst

b"diff --git a/src/plone/app/testing/layers.rst b/src/plone/app/testing/layers.rst\nindex 23c7d9c..49291e8 100644\n--- a/src/plone/app/testing/layers.rst\n+++ b/src/plone/app/testing/layers.rst\n@@ -374,7 +374,8 @@ When the server is torn down, the ZServer thread is stopped.\n     Tear down plone.testing.zope.Startup in ... seconds.\n     Tear down plone.testing.zca.LayerCleanup in ... seconds.\n \n-    >>> conn = urlopen(portal_url + '/folder1', timeout=5) # doctest: +IGNORE_EXCEPTION_DETAIL\n+    >>> import requests\n+    >>> requests.get(portal_url + '/folder1') # doctest: +IGNORE_EXCEPTION_DETAIL\n     Traceback (most recent call last):\n     ...\n-    urllib.error.URLError: <urlopen error [Errno 61] Connection refused>\n+    requests.exceptions.ConnectionError: ...\n"

Repository: plone.app.testing


Branch: refs/heads/master
Date: 2019-02-09T18:50:53+01:00
Author: Jens W. Klein (jensens) <jk@kleinundpartner.at>
Commit: https://github.com/plone/plone.app.testing/commit/a6fde539d143978d8ad1a1c5475874ceb789073b

Merge pull request #56 from plone/workaround

WIP Workaround for https://github.com/plone/Products.CMFPlone/issues/â€¦

Files changed:
M src/plone/app/testing/layers.rst

b"diff --git a/src/plone/app/testing/layers.rst b/src/plone/app/testing/layers.rst\nindex 23c7d9c..49291e8 100644\n--- a/src/plone/app/testing/layers.rst\n+++ b/src/plone/app/testing/layers.rst\n@@ -374,7 +374,8 @@ When the server is torn down, the ZServer thread is stopped.\n     Tear down plone.testing.zope.Startup in ... seconds.\n     Tear down plone.testing.zca.LayerCleanup in ... seconds.\n \n-    >>> conn = urlopen(portal_url + '/folder1', timeout=5) # doctest: +IGNORE_EXCEPTION_DETAIL\n+    >>> import requests\n+    >>> requests.get(portal_url + '/folder1') # doctest: +IGNORE_EXCEPTION_DETAIL\n     Traceback (most recent call last):\n     ...\n-    urllib.error.URLError: <urlopen error [Errno 61] Connection refused>\n+    requests.exceptions.ConnectionError: ...\n"

